scenarioo-server/src/main/java/org/scenarioo/model/configuration/Configuration.java

```diff
@@ -60,6 +60,12 @@
 
 	private String applicationInformation = "";
 
+	/**
+	 * https://github.com/scenarioo/scenarioo/issues/601
```

Comment:
Can you please explain shortly here instead of listing the issue, nobody wants to read that issue anymore, when this is done.